gr-azure-software-radio/python/blob_source.py

```diff
@@ -46,27 +48,17 @@ class BlobSource(gr.sync_block):
     Retry Total: Total number of Azure API retries to allow before throwing an exception. Higher
         numbers make the block more resilient to intermittent failures, lower numbers help to
         more quickly iteratively debug authentication and permissions issues.
+    SigMF Recording: Read in a SigMF recording.  The blob name should not include the .sigmf-data extension,
+        the .sigmf-data and .sigmf-meta will get added automatically.  The sample rate and frequency
+        will be added as stream tags to sample 0.  The datatype specified within the meta file is not
+        used; the datatype is set by the Output Type specified above.  The meta file as a whole is
+        printed to console at the beginning of flowgraph execution.
     """
-    # pylint: disable=too-many-arguments, too-many-instance-attributes, arguments-differ, abstract-method
+    # pylint: disable=too-many-arguments, too-many-instance-attributes, arguments-differ, abstract-method, too-many-locals
     def __init__(self, np_dtype: np.dtype, vlen: int = 1, authentication_method: str = "default",
                  connection_str: str = None, url: str = None, container_name: str = None, blob_name: str = None,
-                 queue_size: int = 4, retry_total: int = 10, repeat: bool = False):
+                 queue_size: int = 4, retry_total: int = 10, repeat: bool = False, sigmf: bool = False):
         """ Initialize the blob_source block
-
```

Comment:
same comment as above. The documentation for python blocks shows up in the docs tab for me. Let's confirm they don't  show up before removing it from the source code.